store/src/java/com/zimbra/cs/session/SessionCache.java

```diff
@@ -42,6 +42,27 @@
  **/
 public final class SessionCache {
 
+    private static final String sRunIdentifier = (System.currentTimeMillis() / 1000) + "." + new java.util.Random().nextInt(100);
```

Comment:
Would a UUID be better? This is probably unique enough in reality.

Reply:
I'm inclined to leave that alone for now although I take your point :-)

Reply:
It was just a thought. I agree this is probably unique enough. If any sessions double up in exactly the same millisecond theoretically the 'nextInt' will not generate the same exact number although I suppose that possibility exists however small.

I am happy for you to leave it alone as well.